# Exam Answers

## 2018 Exam
### Part A
1. base pair differences b/w chimp and human in regions under genetic selection
    - answer = C = 30 000 000
    - human genome = 3 billion base pairs (30 000 000 000)
    - chimp-human difference about 1:100
        - 300 000 000 = 300 million
2. current state of personal genetic testing
    - answer = C = there are serious issues concerning the advice they might provide

3. affinity tag - mass spectrometry protein-protein interaction studies
    - answer = B = typically biased towards stable interactions
    - do not detect rare and common proteins with equal efficacy
    - work for proteins with known and unknown functions
    - co-immunoprecipitation is usually a validation step after for ex. two-hybrid analysis when investigating protein-protein interactions
    - not always able to identify the proteins in a complex

4. Domain accretion describes the
    - answer (OR - E seems more like it)
        - **E = the increase in the number of conserved motifs in protein families in more complex organisms**
        - B = apparent growth of individual conserved domain size in complex organisms
    - domain accretion = The evolution of larger, multidomain proteins by the addition of DNA segments encoding distinct structural domains

5. in genome wide association studies where mutations in many genes may contribute to a phenotype, statistically significant results can be dependent upon
    - answer = E - all of the above
        - distance between SNPs tested
        - number of genes involved
        - sample size
        - allele frequencies

6. There are more than 50 million or more human SNPs, which of the following is true:
    - answer = B = SNPs continue to be found at a high rate
    - we are likely to find more than a few million more
    - we are not likely to fnd all SNPs in human pop
    - rare SNPs are likely major contributor to human phenotype
    - not likely due to sequencing errors but rather error of repair for mutation

7. The small mycoplasma genetalium (parasitic bacterium) genome has about 500 protein coding genes, which is true:
    - answer = A = it is the smallest bacterial genome known
    - answer also B = gene deletion studies show that about 300 of these genes are essential
    - individually deleting the non-essential 200 genes does affect fitness - false because 100 of them are actually non-essential?
    - pelagibacter ubique - one of the smallest known free living bacterium with 1354 protein coding genes

8. A SNP associated with a particular phenotype in GWAS studies is predicted to be surrounded by other SNPs also displaying linkage disequilibrium, which is true
    - answer = C = the results from low recombination rates

9. Homolog
    - answer = C = describing gene sets arising from divergent evolution
    - answer = D = describes members of a multicopy gene family

10. sequence LHRWAP has an E value of about 600 when submitted to BLASTP and the nr protein database, and LAMSLV has an E value of about 5000. No profile for either sequence was found in CDD. The different E values are in part because:
    - answer - E = None of the following:
        - w is a low frequency convserved amino acid in proteins
        - the LAMSLC seq is an important small conserved structural domain in major protein family - false because high E value means it's very likely to occur due to random chance
        - the LHRWAP sequence is likely very rare in proteins
        - the high E value suggest that the LAMSLC and LHRWAP seq represent strongly conserved motifs in many protein families
    - BLASTP = This program, given a protein query, returns the most similar protein sequences from the protein database that the user specifies
    - E-value = describes the number of hits one can "expect" to see by chance when searching a database of a particular size. The lower the E-value, or the closer it is to zero, the more "significant" the match is. Length dependent, short sequences tend to have high E values.
    - CDD = The Conserved Domain Database is a resource for the annotation of functional units in proteins. Its collection of domain models includes a set curated by NCBI, which utilizes 3D structure to provide insights into sequence/structure/function relationships

11. Which BLAST builds profiles?
    - answer = E = PSIBLAST
        - psiblast = position specific blast
            - iterative blast search based on building a PSSM (position specific scoring matrix analogous to a substitution matrix but for each position in the sequence) amongst significant hits with each successive run
            - Thus, PSI-BLAST provides a means of detecting distant relationships between proteins.
            - (PSI)-BLAST is a protein sequence profile search method that builds off the alignments generated by a run of the BLASTp program.
                - first generate multiple alignment of the highest scoring pairs of BLASTP run above threshold and calculates a profile
                - profile is a matrix of scores for each position in the alignment where high conservation score high
                - profile is further used  as substituion matrix to search DB for pattern that matches conservation pattern

12. If you were updating info on a gene annotated as a sequence orphan in the squirrel genome (no apparent orthologues), the best place to look for a potential orthologue is to:
    - answer = D = look in related species with fully sequenced genomes
    - Orthologs are genes that are related by vertical descent from a common ancestor and encode proteins with the same function in different species
    - therefore if you want to find info of a sequence orphan then look for similar species since we want function and orthologues mean similar function
    - sidenote - paralog - gene has evolved by duplication and code for protein with similar but not identical function
13. arabidopsis genome contains approx. how mnay protein coding genes
    - answer = 27â€‰235 therefore = either D (25000) or E (30000)
    - arabidopsis - small flowering plants related to cabbage and mustard

14. A synthetic lethality interactome is described by:
    - answer = D = a scale free network (similar to that of airport USA map)
    -  synthetic lethal screen is a method of isolating novel mutants whose survival is dependent on a gene of interest.

15. Thousand Genome Project - high resolution mapping of human evolutionary constraint, we can conclude that:
    - answer = D = synonymous constraint was common in protein open reading frames
    - the genomes compared gave SUFFICIENT statistical power to draw conclusions
    - about 5.5% of the mammalian genome is under evolutionary selection (fixing the bad)
    - almost all of the contrained elements found in non coding genes (intronic, intergenic)
    - synonymous mutations - mutation that results in same amino acid -- those that do not cause an amino acid change
    - Synonymous constraint {constrained symmetry} elements (SCEs) are protein-coding genomic regions with very low synonymous mutation rates (high AA changes - more mutations) believed to carry additional, overlapping functions
    - protein open reading frame = reading frame that has the ability to be translated (Start -> Stop)

16. Some of the most powerful multiple alignments tools search by:
    - answer = B = I think the most powerful M/A tool is CLUSTALW, which uses (optimal pairwise alignments (global and local))
    - COBALT - is another alternative - COBALT does progressive multiple alignment of protein sequences. The alignment is aided by a collection of pairwise constraints derived from conserved domain database

17. miRNA genes
    - answer = C = participate in post-transcriptional regulation of mRNAs

18. Using affinity chromatography (AC) - mass spectrometry (MA), if we define a set of 6 proteins in humans as interacting with each other to form a core module within the overall protein interactome then we might expect that:
    - answer = D = invidual gene deletions of the corresponding six genes may share a common phenotype (they are all related)
    - not yeast 2-hybrid analysis - because Y2H - checks for interactions between two proteins using Bait and prey

19. DNA complexity refers to
    - answer = B = length of genome minus the introns
    - genome contains many additional feature such as UTRs, regulatory elements (intergenic DNA), non-protein-coding genes and chromosomal structural elements (not introns {functionalRNA, allows for alternative splicing})

20. In running a homology modelling program such as Swiss Model to model 3D structure of a new protein, a sequence is fitted to a known set of 3D coordinates
    - answer = D = the fit to the crystal coordinates tends to be better where the similarity score between the new sequence and the protein from which the 3D coordinates were derived is high
    - SWISS model:
        - start with a blast search to find orthologues and then models your primary sequence into coordinates of a similar protein with known structure
        - the effectiveness of the technique can be assessed by asking how closely the sequences actually match known sets of protein 3D coordinates

21. Transcriptional profiling and matching the patterns of gene expression for different mutants vs. drug treatments is most useful for:
    - answer = A = inferring the molecular target of the drug
    - how do we find the targets of miRNAs?
        - gene KO (mutant) plus transcriptional profile and phenotype
        - gene overexpression plus transcriptional profile and phenotype

22. Competitive hybridization:
    - answer = A = controls for some problems associated with uneven hybridization to an array
    - Competitive hybridization is a simple yet powerful method that was developed to screen cDNA libraries for differentially regulated genes.
        - allows for the quick comparison checking for gains or losses of either whole/sub chromosomal regions

23. Doolittle's rules include
    - answer = E = all of the included
        - sequences > 100 AA with > 25% identity and suitible gapping are likely related
        - sequences <100 AA with >25% identity and no gaps are likely related
        - sequences with 15-25% identity may be related but not necessarily so
        - sequences <15% identical are probably not related
        - if an alignment has >1 gap per 20 aa, suspicious
        - if alignment drastically changes when gap penalties are changed by small amounts then alignment is probably not significant
24. SNPs in the human genome are:
    - answer = E = SNPs are at lower frequency in coding regions
    - occur in non-coding regions more frequently
        - hence not always 'breaking' coding regions
    - can be predicted using microsatellites
    - most associated with synonymous mutations
    - not clustered at sites important for protein structure
    - not randomly distribution
    - have not all been found

25. In comparing genes between chimps and humans we can:
    - answer = B = find some examples of strong positive selection (brain dev?)
    - not easily detect genes that make chimps and humans different
    - CAN find example of negative (purify) selection
    - not find wide divergence in large number of genes
    - CAN find examples of gene loss in one or the other

### Part B - T/F
26. Similarity scores are not depending on the substitution matrix used
    - F -- similarity scores need the sub matrix

27. Bait-prey pair in 2-hybrid analysis finds physically interacting DNA sequences
    - F -- uses reporters, and finds physically interacting proteins

28. Many polymorphisms are common and therefore probably are young in terms of when they entered the human population
    - F -- usually a SNP is removed and if it remains it's favoured and therefore passed on

29. If one is doing environmental genomics by PCRing genes from environmental samples it is best to use primers directed to a highly variable gene so that as much diversity as possible can be found
    - F -- want conserved for multi-species converage
    - design PCR primers to the most conserved domains to allow amplification of a bit of DNA gene from all species
    - PCR primers should flank variable regions for greatest sensitivity

30. We are now confident that in most cases we can predict the 3D structure of a protein based solely on its primary sequence
    - F -- can use SWISS model which first uses BLASTP but need validation

31. Only about 2% of the human genome sequence is under genetic constraint
    - F -- 5% of the human genome is under genetic constraint

32. To be useful a transcriptional profile must be reproducible and in some way reflect the underlying physiological state of the state
    - T -- miRNA - find targets of miRNA; transcriptional profile + phenotype

33. The use of strongly predictive genetic tests is still limited to a small number of highly hereditary diseases
    - T -- predictive genetic tests only work on small nuber of highly hereditary diseases

34. Protein coding regions make up about 5% of the human genome
    - F -- humans protein coding regions make up 1.5%

35. the primary sequence databases are no longer growing at an exponential rate because all of the major experimental organisms have now been sequenced
    - F - GenBank is tappering but WGS (whole genome shotgunning) is still exponential

36. RNA recoding on conserved sites by adenine deaminases is common in human cells
    - F -- frequencies of RNA recoding in humans is 3%; most is on non-conserved sites

37. The doolittle diagram shows the level of identity between two proteins needed to infer homology. Since this value drops with length, very long proteins (ex. over 700aa) only need 10% identity to be considered homologous
    - F -- anything less than 15 is not homoloous, 15-25% is maybe, <100aa & >25% & no gaps is yes, >100aa & >25% & suitable gaps is yes homologous

38. In an interaction map based on synthetic lethal data,the lines joining genes represent protein-protein interactions
    - F -- idk why

39. Epigenetic methylation of histones activates gene expression
    - T & F -- some are activated and some deactivated (from chart most are activated)
40. lnRNA (lncRNA - long non-coding RNA) are rare in human genome
    - F -- ~58000 lncRNAs in humans whereas only ~22000 protein coding genes

41. miRNA can both activate and repress protein translation
    - T

## 2017 Exam
### Part A - T/F
1. Complexity of human genome is 3 billion base pairs
    - T -- to the size part but genome size does not define complexity

2. Most human genes have an orthologue in capybara (rodent)
    - F -- manual checking of orthology b/w rodent and human?

3. A large percentage of human transcription start sites do not align with the mouse genome
    - T

4. Hi-C analysis can be used to discover associations between enhancer elements and promoters
    - answer = T? -- may not be tested
    - Hi-C (all-vs-all) uses high throughput sequencing to find the nucleoide sequence of fragments, all pairwise interactions between fragments are tested
    - 3C (one vs. one) - quantifies interacions between a single pair of genomic loci - can be used for promoter-enhancer interaction

5. Species can be defined by the discovery of multiple isolates of a novel sequence from the environment differeing from all previously known sequences by more than 1% divergence
    - T -- can't find evidence

6. lncRNA genes are less common than protein coding genes in humans
    - False -- more common than protein coding genes

7. Methylation of CpG islands activates gene expression
    - F --repression

8. 

### Part B - MC